src/condor_negotiator.V6/matchmaker.cpp

```diff
@@ -3688,6 +3688,54 @@ sumSlotWeights(ClassAdListDoesNotDeleteAds &startdAds, double* minSlotWeight, Ex
 	return sum;
 }
 
+
+bool
+Matchmaker::TransformSubmitterAd(classad::ClassAd &ad)
```

Comment:
Especially with the death of compat classad, I'd prefer to see the unqualified "ClassAd" everywhere, especially in interfaces.

Reply:
Until all uses of ClassAd are fully qualified, it is asking for trouble that some interfaces here are fully qualified and some not.